examples/python/device_selection.py

```diff
@@ -25,8 +25,8 @@ def create_default_device():
     Create default SyclDevice using `sycl::default_selector`.
 
     Device created can be influenced by environment variable
-    SYCL_DEVICE_FILTER, which determines SYCL devices seen by the
-    SYCL runtime.
+    ONEAPI_DEVICE_SELECTOR, which determines SYCL devices seen by the
+    oneAPI DPC++ runtime.
```

Comment:
Should be `SYCL runtime` replaced with `oneAPI DPC++ runtime` everywhere? Still present in many files.

Reply:
Probably yes, it is more precise, as we do not presently support other SYCL implementations, but I would defer that change to a different PR.